### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by philhallbio**
## Description

New wording for local npm install and lint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Looking for something specific? 🔍
 
 Initially you will need to install [Ruby <2.7](https://www.ruby-lang.org/en/documentation/installation/) and [bundler](https://bundler.io/) with `gem install bundler`.
 
-Run `bundle config set --local path 'vendor/bundle'` to make the bundle install everything locally (so no sudo is required) and install dependenies locally with `bundle install` and npm dependencies for lint and format with `npm i`.
+Run `bundle config set --local path 'vendor/bundle'` to make the bundle install everything locally (so no sudo is required) and install dependenies locally with `bundle install` and npm dependencies for lint and format with `npm i --include=dev`.
```
Comment (ChasNelson1990) [inline README.md]: We don't need bundle or ruby at all now :-), just the npm install.

File: README.md [hunk 2]
```diff
@@ -50,11 +50,17 @@ Don't forget to regulary update dependenies with `bundle update` and `npm update
 
 As a standard, all code contributions should be linted with [ESLint](https://eslint.org/) using `.eslintrc.js` and formatted with [Prettier](https://prettier.io/). **Note:** HTML + CSS, mark-up and mark-down code are exemptions and should be formatted using [Prettier](https://prettier.io/) but do not need to be linted.
 
+We recommend that you use globbing to lint all markdown files in the project. To do this, please replace the line "lint:md" in the package.json file with "lint:md": "npx markdownlint-cli --ignore node_modules ./*/.md".
```
Comment (ChasNelson1990) [inline README.md]: Why don't you actually just commit that change?

File: README.md [hunk 3]
```diff
@@ -50,11 +50,17 @@ Don't forget to regulary update dependenies with `bundle update` and `npm update
...
+
 `npm run lint` will lint the codebase.
 `npm run format` will format the codebase.
 
 Our GitHub Actions will also lint any pull requests before they're merged.
 
+## Build Testing
+
+Once code alterations have been made, please ensure the validity and stability of the code. We recommend using [Docusaurus](https://docusaurus.io/) using command 'npm run build' or 'npm run start' for continuous visualisation.
```
Comment (ChasNelson1990) [inline README.md]: We don't recommend using docusaurus.... we do use docusaurus lol.

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]

**Review by joshuajames-smith — CHANGES_REQUESTED** [review]
One other small change I think we also need to make is on line 7.
Currently is:
```
This repository contains the Open Source code for [gliff.ai](https://gliff.ai)’s DOCUMENT supporting library (built using the [Just The Docs](https://pmarsceill.github.io/just-the-docs/) Jekyll template).
```

Change it to:
```
This repository contains the Open Source code for [gliff.ai](https://gliff.ai)’s DOCUMENT supporting library (built using [Docusaurus](https://docusaurus.io)).
```

Otherwise, I think this is good to go once changes are made!

**Review by joshuajames-smith — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**